src/lava/lib/dl/slayer/io.py

```diff
@@ -564,8 +569,17 @@ def tensor_to_event(spike_tensor, sampling_time=1):
 
     >>> td_event = tensor_to_Event(spike)
     """
-    if spike_tensor.ndim == 3:
+    if isinstance(spike_tensor, torch.Tensor):
```

Comment:
Can you add a unittest to test whether the function works consistently for numy and torch?

Reply:
I pushed a commit adding a unittest as requested